### PR TITLE
[vcpkg-ci-boost] All triplets

### PR DESCRIPTION
**Pull request by dg0yt**

### CODE REVIEW

File: scripts/ci.baseline.txt [hunk 1]
```diff
@@ -1214,6 +1214,8 @@ vcpkg-ci-arrow:x64-osx=pass
 vcpkg-ci-arrow:x64-linux=pass
 vcpkg-ci-boost:x64-linux=pass
 vcpkg-ci-boost:x64-osx=pass
+vcpkg-ci-boost:x64-arm=pass
```
Comment (Osyotr) [inline scripts/ci.baseline.txt:1217]: `arm64-uwp`?

File: ports/mapnik/vcpkg.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "mapnik",
   "version-date": "2022-01-28",
-  "port-version": 2,
+  "port-version": 3,
   "description": "Mapnik is an open source toolkit for developing mapping applications.",
   "homepage": "https://github.com/mapnik/mapnik",
   "license": "LGPL-2.1-only",
-  "supports": "!(static & windows)",
+  "supports": "!(static & windows) & !uwp",
```
Comment (dg0yt) [inline ports/mapnik/vcpkg.json:8]: With this change, mapnik is `n/a` in x64-uwp. But `vcpkg ci` still pulls in `boost-regex[icu]`, resulting in `boost-regex` not being buildable in x64-uwp CI.

Comment (dg0yt) [reply ports/mapnik/vcpkg.json:8]: https://github.com/microsoft/vcpkg-tool/pull/846

**Comment by dg0yt** [comment]
vcpkg-ci-boost:arm64-windows should build, not cascade, according to depend-info and dry-run install. 
Still something pulling in an unsupported dependency?

**Comment by JonLiu1993** [comment]
Pinging @dg0yt for response. Is work still being done for this PR?

**Comment by dg0yt** [comment]
Yes, this is to be finished.

**Comment by dg0yt** [comment]
The following ports request boost-mpi which transitively relies on msmpi:arm64-windows which is set to "skip" in vcpkg CI:
boost-graph-parallel
boost-property-map-parallel

If there is a chance for msmpi to work on arm64 (as indicated by "skip"), the the only solution is to hide the boost-mpi dependency behinde a feature, as done in other boost ports.

**Review by BillyONeal — APPROVED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Not every day we get to merge things from almost a year ago, thanks!

**Comment by dg0yt** [comment]
There are still some quite useful PRs waiting to be finished (or accepted)...

**Pull request merged**